site/docs/configuration.md

```diff
@@ -40,6 +40,9 @@ Iceberg tables support table properties to configure table behavior, like the de
 | write.parquet.dict-size-bytes      | 2097152 (2 MB)     | Parquet dictionary page size                       |
 | write.parquet.compression-codec    | gzip               | Parquet compression codec                          |
 | write.parquet.compression-level    | null               | Parquet compression level                          |
+| write.parquet.bloom-filter-enabled | false | Whether to enable writing bloom filter; It is also possible to enable it for some columns by specifying the column name within the property followed by # |
```

Comment:
> I also think that we need to more carefully consider how to configure Parquet's bloom filters. I would expect what you've added here as table properties to be column specific. Why did you choose global settings. Does this create a bloom filter with the same NDV for all columns?

@rdblue  Yes, `write.parquet.bloom-filter-enabled` and `write.parquet.bloom-filter-expected-ndv` both support column specific settings. We can set `write.parquet.bloom-filter-enabled#user_id=true` and `write.parquet.bloom-filter-expected-ndv#user_id=1000` to just enable bloom filter for column `user_id` with NDV 1000.
I'll make the doc more complete in new PRs.